app/models/cache/Answers.scala

```diff
@@ -66,6 +66,8 @@ object ArrivalAnswers {
       ArrivalAnswers(consignmentReferences = Some(ConsignmentReferences(ConsignmentReferenceType.M, ucrBlock.ucr)))
     case Some(ucrBlock) if ucrBlock.ucrType.equals(Ducr.codeValue) =>
       ArrivalAnswers(consignmentReferences = Some(ConsignmentReferences(ConsignmentReferenceType.D, ucrBlock.ucr)))
+    case Some(ucrBlock) if ucrBlock.ucrType.equals(DucrPart.codeValue) =>
+      ArrivalAnswers(consignmentReferences = Some(ConsignmentReferences(ConsignmentReferenceType.DP, ucrBlock.ucr)))
```

Comment:
Would all of these DUCR Part answers be safer if they were using the ucrBlock.fullUcr method?    I know they currently return the same thing but if we decide to change the UcrBlock implementation to store the partId separately then this code here won't need to change.

Applies to all changes to this file

Reply:
Yeah, pushed added this to review fixes commit. Good catch